sdk/component/conf.go

```diff
@@ -19,10 +19,15 @@ const (
 	errReasonCannotBeNil      = "cannot be nil"
 
 	// Env vars.
-	envVarKeyComponentName   = "SMITHY_COMPONENT_NAME"
-	envVarKeyWorkflowID      = "SMITHY_WORKFLOW_ID"
+	// -- BASE
+	envVarKeyComponentName = "SMITHY_COMPONENT_NAME"
```

Comment:
this should be defined by the component itself in the entrypoint not an environment variable

Reply:
Why is that? Can you elaborate?

Reply:
Took this offline - I can tackle it in a separate small pr if that's ok

Reply:
https://linear.app/smithy/issue/OCU-265/leverage-component-name-as-argument-for-components